tests/admin-mode_test.go

```diff
@@ -98,38 +98,28 @@ var _ = Describe("BDD of operator reconcile resiliency check", func() {
 			Expect(err).To(BeNil(), "Failed to create deployment")
 			fmt.Println("Test Application is created")
 
-			//Removing pod delete Experiment if it already exists in litmus ns
-			By("Experiment Cleanup")
```

Comment:
Why did you remove this?
Instead of changing the experiment name to a different name and create it again. This approach is better where you delete the older experiment CR and create a new with the latest changes cuz for us experiment name is a universal truth? what do u think?

Reply:
The issue with this is:
if it doesn't get a chaos engine with the name `engine` it will fail.
or
if it doesn't get chaos experiment `pod-delete`. `kubectl delete chaosexpeirment pod-delete` will fail.

So It is now dependent on `pod-delete` Exp. if pod delete is not present then this test will fail at those commands

Reply:
That's why changing the name.